services/web/client/source/class/qxapp/component/widget/FileTreeItem.js

```diff
@@ -62,33 +63,43 @@ qx.Class.define("qxapp.component.widget.FileTreeItem", {
         flex: 1
       });
 
+      // Add size
+      var sizeWidget = new qx.ui.basic.Label().set({
+        width: 70,
+        maxWidth: 70,
```

Comment:
in general why do you have fixed values? are these pixels, percents or points?

Reply:
Those are pixels.
In this case we fix the width to have exactly the same layout in each tree entry.

Reply:
Just a bit wondering on how things are going when on different screen sizes. In Qt, you can defined fixed size without defining their absolute number, such that they can adapt to different resolutions.